### PR TITLE
Features/bitcoin d

### PR DESCRIPTION
**Pull request by ghost**
temporary save

### CODE REVIEW

File: packages/blockchain-link/report.20200305.113944.3450.0.001.json [hunk 1]
```diff
@@ -0,0 +1,764 @@
+
```
Comment (szymonlesisz) [inline packages/blockchain-link/report.20200305.113944.3450.0.001.json]: remove this file

File: packages/blockchain-link/src/ui/index.ui.ts [hunk 2]
```diff
@@ -2,10 +2,13 @@
 import BlockbookWorker from '../workers/blockbook/index';
 // @ts-ignore no default export
 import RippleWorker from '../workers/ripple/index';
+// @ts-ignore no default export
+import BitcoindWorker from '../workers/bitcoind/index';
 
 import CONFIG from './config';
 import BlockchainLink from '../index';
 import { getInputValue, fillValues, onClear } from './utils';
+import { common } from 'ripple-lib/dist/npm/ledger/utils';
```
Comment (szymonlesisz) [inline packages/blockchain-link/src/ui/index.ui.ts:11]: unused import

File: packages/blockchain-link/report.20200305.113944.3450.0.001.json (hunk 1 above)
Comment (ghost) [reply packages/blockchain-link/report.20200305.113944.3450.0.001.json]: deleted

File: packages/blockchain-link/package.json [hunk 3]
```diff
@@ -38,14 +38,19 @@
...
+    "@trezor/utxo-lib": "^0.1.0",
     "babel-eslint": "^10.0.3",
     "babel-jest": "^25.1.0",
     "babel-loader": "^8.0.6",
     "bignumber.js": "^9.0.0",
+    "bitcoind-rpc": "^0.8.1",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-loader": "^3.0.3",
     "eslint-plugin-jest": "^23.7.0",
+    "golomb": "^1.2.0",
```
Comment (szymonlesisz) [inline packages/blockchain-link/package.json]: is this needed?

File: packages/blockchain-link/package.json [hunk 4]
```diff
@@ -38,14 +38,19 @@
...
     "html-webpack-plugin": "^3.2.0",
+    "json-rpc-client": "^0.2.2",
+    "node-bitcoin-rpc": "^1.1.3",
```
Comment (szymonlesisz) [inline packages/blockchain-link/package.json]: looks unused

**Comment by ahmad380360** [comment]
good

**Comment by szymonlesisz** [comment]
closing. continuation on https://github.com/trezor/trezor-suite/tree/feature/trezord

**Pull request closed without merging**